webdriver/newsession/basic.py

```diff
@@ -0,0 +1,57 @@
+# META: timeout=long
```

Comment:
I don’t think this filename is great. Maybe call it webdriver/newsession/defaults.py or something similar.